server/src/server.ts

```diff
@@ -152,9 +74,11 @@ class CustomErrorListener {
 
 	private _getErrorMessage (msg: string, offendingSymbol: any) : string {
 		let text: string = offendingSymbol.source[1].strdata; 
+		let errorPiece: string = text.substring (32, text.length - 1); 
 		let start: number = offendingSymbol.start; 
 		console.log (offendingSymbol); 
 		console.log (text); 
+		console.log (errorPiece); 
```

Comment:
Do we need both? Rather, do you have the sample output on the log when an error occurs? That would help

Reply:
The variable 'text' is the whole text in the editor, while 'errorPiece' stands for the piece of text where error occurs.

Reply:
Does the user see these texts in the Problems tab? Does it make sense to show both without any other text around it (it looks like we are printing them to the log are the on separate lines, one next to each other?)

Reply:
The Problems tab will only show the error message, which is 'msg' here. I used 'errorPiece' because the original message contains the information about what the error text is. And I would like do some further parsing work with it.

Reply:
I printed them just because of debugging. Do I need to delete these 'console.log' code?